sdk/appcenter/src/main/java/com/microsoft/appcenter/http/TLS1_2SocketFactory.java

```diff
@@ -32,6 +33,11 @@
      */
     private static final String[] ENABLED_PROTOCOLS = { TLS1_2_PROTOCOL };
 
+    /**
+     * Socket timeout
+     */
+    private static final int SO_TIMEOUT = 5000;
```

Comment:
this is not the right place, this code is executed only on specific versions of Android.
Please look where we actually configure the url connection, we already have constants for connect and read timeouts.